examples/ceed/ex2_surface.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Apr 18 11:41:54 2025
+
+@author: surindersinghchhabra
+"""
+
+import numpy as np
+from math import fabs
+import sys
+import argparse
+from libceed import Ceed, _ceed_cffi
+from libceed.ceed_elemrestriction import ElemRestriction
+
+
+# Import CEED constants from the compiled CFFI bindings
+lib = _ceed_cffi.lib
```

Comment:
You should not be using the raw C bindings.

The Python tutoral notebooks show using the native Python interface: https://github.com/CEED/libCEED/tree/main/examples/python